### PR TITLE
Improving the sound for security

### PR DESCRIPTION
**Pull request by Klaypexx**
## About the PR
- Changing sound to jackboots and combat boots (add sound heavystep1.ogg, heavystep2.ogg,heavystep3.ogg ,heavystep4.ogg). 

**Media**

https://user-images.githubusercontent.com/68664576/232811864-605b1055-622d-46eb-be45-374bcbb856a9.mov

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Klaypexx
- tweak: Jackboots and Combat boots sound changed to unique

### CODE REVIEW

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
The sound for the combat boots in this pr is a copyrighted Half Life 2 sound effect and cannot be used. Please change it.

**Review by keronshb — APPROVED** [review]
> The sound for the combat boots in this pr is a copyrighted Half Life 2 sound effect and cannot be used. Please change it.

~~Please address these changes before this PR can be merged.~~

Oops hit the wrong button here ignore this approval.

**Review by keronshb — CHANGES_REQUESTED** [review]
> The sound for the combat boots in this pr is a copyrighted Half Life 2 sound effect and cannot be used. Please change it.

Please address these changes before this PR can be merged.

File: Resources/Audio/Effects/Footsteps/sources.txt [hunk 1]
```diff
@@ -11,3 +11,5 @@ grass1.ogg and grass2.ogg are taken from https://freesound.org/people/D001447733
 grass3.ogg and grass4.ogg are taken from https://freesound.org/people/jevans27/sounds/563529/ by jevans27 licensed under CC-0 and split/modified by mirrorcult
 
 snowstep1.ogg, snowstep2.ogg, snowstep3.ogg, and snowstep4.ogg are taken from https://freesound.org/people/alexo400/sounds/543385/ by alexo400 licensed under CC-0 and split by EmoGarbage404
+
+heavystep1.ogg, heavystep2.ogg, heavystep3.ogg, heavystep4.ogg are taken from https://mp3crown.org/downloadmp3/oeh0x71lkbx_heavy-footstep-sound-effect-hd-hq.html
```
[the remark this answers is not in the document]
Comment (Klaypexx) [reply Resources/Audio/Effects/Footsteps/sources.txt:15]: > > The sound for the combat boots in this pr is a copyrighted Half Life 2 sound effect and cannot be used. Please change it.
> 
> Please address these changes before this PR can be merged.

Okay, no problem

**Comment by Klaypexx** [comment]
> > The sound for the combat boots in this pr is a copyrighted Half Life 2 sound effect and cannot be used. Please change it.
> 
> ~Please address these changes before this PR can be merged.~
> 
> Oops hit the wrong button here ignore this approval.

Heavystep1-4 i completely download from youtube (https://m.youtube.com/watch?v=xbkl17x0heo) , the author under the video wrote that the sound, I quote « Feel free to use, i altered the sound of this youtubers sound effect to make it sound more heavy and intense. Enjoy!»

As for combat boots, I will try to find a license for it or find a more suitable sound.

**Pull request closed without merging**